data/net/minecraft/world/entity/ai/behavior/Behavior.mapping

```diff
@@ -23,7 +23,7 @@ CLASS net/minecraft/world/entity/ai/behavior/Behavior
 		ARG 1 owner
 	METHOD start (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/world/entity/LivingEntity;J)V
 		ARG 1 level
-		ARG 2 entity
+		ARG 2 memoryHolder
```

Comment:
Why `memoryHolder` and not `entity`?

Reply:
I'm not sure why Lanse chose `memoryHolder` here. I would agree `entity` make more sense,